cpp/src/ice2slice/Gen.cpp

```diff
@@ -350,6 +349,8 @@ Gen::generate(const UnitPtr& p)
 
     TypesVisitor typesVisitor(_fileBase, outputVisitor.modules());
     p->visit(&typesVisitor);
+
```

Comment:
I am not following.

Is the TypesVisitor creating files? And if yes, why can't it add a new line to each file it creates on its own?

Reply:
I think destructor or some other code in the Visitor would be nicer than this explicit call.